README.md

```diff
@@ -61,20 +61,15 @@ Running Masonry's integration tests. This will leave lots of Firefox processes h
 
 When a release will cause breaking changes — in usage or in typing — we provide a codemod to ease the upgrade process. Codemods are organized by release in `/packages/gestalt-codemods`. We recommend using [jscodeshift](https://github.com/facebook/jscodeshift) to upgrade.
 
-### Prerequisite:
-Install `jscodeshift` globally if you haven't already.
-```bash
-yarn global add jscodeshift
```

Comment:
`jscodeshift` is now installed as a dev dependency and we wrap it with `yarn codemod` to ensure the correct `jscodeshift` version.